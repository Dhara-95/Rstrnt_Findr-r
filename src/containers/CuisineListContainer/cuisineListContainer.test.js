import React from "react";
import renderer from "react-test-renderer";
import { CuisineListContainer } from "./index";

it("displays the Cuisine List Container component correctly", () => {
  const tree = renderer.create(<CuisineListContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
