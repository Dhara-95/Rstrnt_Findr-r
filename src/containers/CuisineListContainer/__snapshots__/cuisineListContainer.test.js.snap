// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Cuisine List Container component correctly 1`] = `
<div
  className="py-8"
>
  <h2
    className="text-2xl text-grey p-6"
  >
    Find By Cuisine
  </h2>
  <div
    className="image-container text-grey flex flex-col items-center justify-center lg:flex-row xl:flex-row 2xl:flex-row"
  >
    <div
      className="w-60 h-60 bg-grey rounded-3xl text-black m-4"
    >
      image 1
    </div>
    <div
      className="w-60 h-60 bg-grey rounded-3xl text-black m-4"
    >
      image 2
    </div>
    <div
      className="w-60 h-60 bg-grey rounded-3xl text-black m-4"
    >
      image 3
    </div>
    <div
      className="w-60 h-60 bg-grey rounded-3xl text-black m-4"
    >
      image 4
    </div>
  </div>
</div>
`;
